src/autocomplete.js

```diff
@@ -50,6 +58,13 @@ class Autocomplete extends Component {
   input = null
   items = []
 
+  getState() {
+    return Object.keys(this.state).reduce((state, key) => {
+      state[key] = this.props[key] == null ? this.state[key] : this.props[key]
+      return state
+    }, {})
+  }
```

Comment:
When I saw this, I was first wondering if there was maybe an elegant way of generally defaulting to `state` if a prop wasn't available in `props` and what I had in mind could be something like this:

```js
getPropsOrState() {
  return {
    ...state,
    ...props
  }
}
```
This nicely overwrites with `props` if defined (the details are a little different to this method, because if a prop is available and set to `null` it would still overwrite the state, but I believe that's what a user would expect)